frontend/src/plusButton/PlusButton.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import Fab from "@material-ui/core/Fab";
+import AddIcon from "@material-ui/icons/Add";
+import "./PlusButton.css";
+
+interface IPlusButtonProps {
+  isVisible: boolean;
```

Comment:
Let's add a prop `onClick` which takes a function. That function gets called in `plusButtonClicked()`